### DISCUSSION TITLE
Increase the vibration amplitude of the controller or can adjust it custom

### DISCUSSION THREAD

**Discussion by Falleninmemary** [darthcloud/BlueRetro#935] [Ideas]
I use ps3 and ps4 controller on my ps2， The vibration intensity is not as strong as the PS2 controller，I hope if can Increase the vibration amplitude of the controller or can adjust it custom.

**Comment by darthcloud** [comment]
Rumble is really primitive right now in BlueRetro. This is why it's off by default.

I do plan to fix this, this year.
